src/inference/src/dev/core_impl.cpp

```diff
@@ -342,11 +334,12 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::compile_model(ov::Plugin& plugin,
         manager.run_passes(cloned_model);
         prepared_model = cloned_model;
     }
-
+    auto _config = config;
```

Comment:
```suggestion
    auto& _config = config;
```

Reply:
`config` is const, but `_config` will be modified.